java/fury-core/src/main/java/org/apache/fury/config/FuryBuilder.java

```diff
@@ -400,7 +400,7 @@ private void finish() {
       if (deserializeNonexistentClass == null) {
         deserializeNonexistentClass = false;
       }
-      if (scopedMetaShareEnabled != null) {
+      if (scopedMetaShareEnabled) {
```

Comment:
This will throw null pointer exception, the default value is null

Reply:
Fixed.